### PR TITLE
Renamed button from 'Add COVID Suspect/Patient' to 'Add Patient'

### LINKED ISSUE #2393: "Add Patient": Patient Registration
The patient registration form at https://care.coronasafe.in/facility/df1a22fb-1875-4949-af85-15d1917b8bfd/patient

has "Add COVID Suspect/Patient" button to save details. Rename this button to "Add Patient"

### PR DESCRIPTION
**Pull request by GokulramGHV**
Fixes #2393 

Renamed the "Add Covid Suspect/Patient" button in the patient registration form to "Add Patient".

Before:
image

After:
image

### CODE REVIEW

**Review by tomahawk-pilot — APPROVED** [review]

**Comment by nihal467** [comment]
sokfd

@gigincg @GokulramGHV this also needs to be changed right?

**Comment by nihal467** [comment]
test approved

**Review by gigincg — CHANGES_REQUESTED** [review]
@GokulramGHV You cannot use the `eslint-disable-next-line`. You need to fix them!

**Comment by GokulramGHV** [comment]
@gigincg I've made the requested changes.

File: src/Components/Patient/PatientRegister.tsx [hunk 1]
```diff
@@ -25,11 +25,11 @@ import {
   GENDER_TYPES,
   MEDICAL_HISTORY_CHOICES,
   TEST_TYPE,
-  DESIGNATION_HEALTH_CARE_WORKER,
+  // DESIGNATION_HEALTH_CARE_WORKER,
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 2]
```diff
@@ -25,11 +25,11 @@ import {
...
   VACCINES,
 } from "../../Common/constants";
 import countryList from "../../Common/static/countries.json";
-import statesList from "../../Common/static/states.json";
+// import statesList from "../../Common/static/states.json";
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 3]
```diff
@@ -66,9 +66,9 @@ const PageTitle = loadable(() => import("../Common/PageTitle"));
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const debounce = require("lodash.debounce");
 
-const placesList = countryList.concat(
-  statesList.filter((i: string) => i !== "Kerala")
-);
+// const placesList = countryList.concat(
+//   statesList.filter((i: string) => i !== "Kerala")
+// );
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the unused import and do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 4]
```diff
@@ -103,7 +103,7 @@ const diseaseStatus = [...DISEASE_STATUS];
 const bloodGroups = [...BLOOD_GROUPS];
 
 const testType = [...TEST_TYPE];
-const designationOfHealthWorkers = [...DESIGNATION_HEALTH_CARE_WORKER];
+// const designationOfHealthWorkers = [...DESIGNATION_HEALTH_CARE_WORKER];
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

File: src/Components/Patient/PatientRegister.tsx [hunk 5]
```diff
@@ -855,12 +853,12 @@ export const PatientRegister = (props: PatientRegisterProps) => {
...
-    const { checked, name } = e.target;
-    form[name] = checked;
-    dispatch({ type: "set_form", form });
-  };
+  // const handleCheckboxFieldChange = (e: any) => {
+  //   const form = { ...state.form };
+  //   const { checked, name } = e.target;
+  //   form[name] = checked;
+  //   dispatch({ type: "set_form", form });
+  // };
```
Comment (khavinshankar) [inline src/Components/Patient/PatientRegister.tsx]: do remove the comment

**Review by khavinshankar — CHANGES_REQUESTED** [review]
please do work on the requested changes,

**Note:**

- _the unused variables, functions, interfaces and imports can be removed instead of commenting them out_
- _do prioritize on fixing an eslint issue instead of using eslint disable_

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**